git.go

```diff
@@ -384,7 +384,14 @@ func (s *GitRepo) ExportDir(dir string) error {
 	if err != nil {
 		return NewLocalError("Unable to export source", err, string(out))
 	}
+
 	// and now, the horror of submodules
+	out, err = s.RunFromDir("git", "submodule", "update", "--init", "--recursive" )
+	s.log(out)
+	if err != nil {
+		return NewLocalError("Error while updating submodule sources", err, string(out))
+	}
```

Comment:
Nope. Good catch!
Just pushed new revision without that -
please take another look when you get a chance -
and Thanks!